pkg/agent/agent.go

```diff
@@ -105,9 +105,6 @@ func agentToken(ctx context.Context, agentNamespace, controllerNamespace string,
 		&v1.Namespace{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: agentNamespace,
-				Annotations: map[string]string{
```

Comment:
I wonder if this should be a label instead.

The [clean up controller](https://github.com/rancher/fleet/blob/master/pkg/controllers/cleanup/handler.go#L117) will purge resources and namespaces when the "managed" label is "true".

If this namespace has no managed=true label, it will be left behind. Actually, it should already be left behind, because the label is set as an annotation instead.